app/scripts/modules/core/src/pipeline/config/templates/templatePlanErrors.component.ts

```diff
@@ -0,0 +1,8 @@
+import { module } from 'angular';
+import { react2angular } from 'react2angular';
+
+import { TemplatePlanErrors } from 'core/pipeline/config/templates/TemplatePlanErrors';
+
+export const TEMPLATE_PLAN_ERRORS = 'spinnaker.templatePlanErrors.component';
```

Comment:
should have been `spinnaker.core.templatePlanErrors.component`, but certainly won't break anything